PyWinTypes error on High-Resolution timer creation and/or start on Win10 LTSC

### Describe the bug
On _Win10 LTSB 1603_ High Resolution timer creation fails with `pywintypes.error 87 (The parameter is incorrect)`, OSError does not catch this. #1449 
[High-Res timers](https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-createwaitabletimerexw#parameters
) were introduced in version 1803 so the fail is understandable, but the exception handling is not working.

On _Win10 LTSC 1809_ the timer creation does not throw an error, but the timer start is failing with error 87.
After some debugging, it is clear that the `CREATE_WAITABLE_TIMER_HIGH_RESOLUTION` flag is the issue. With `CREATE_WAITABLE_TIMER_MANUAL_RESET` the creation and starting is OK.
The high-res timer should be supported at this system version, I haven't found any good reason why could this fail.

<!-- A clear and concise description of what the bug is. -->


### To Reproduce
Try to initialize a ThreadBasedCyclicSendTask with Windows Events.

<!-- Steps to reproduce the behavior. Please add longer code examples below. -->


### Expected behavior
Timer creation/start not to fail. In case of unsupported system version, fall back to normal timer usage.
<!-- A clear and concise description of what you expected to happen. -->


### Additional context

OS and version: Windows 10 LTSB 1603, Windows 10 LTSC 1809
Python version: 3.10.9
python-can version: 4.2.2
python-can interface/s (if applicable): Vector

<!-- Add any other context about the problem here. -->

<details><summary>Traceback and logs</summary>

```
    def start(self) -> None:
        self.stopped = False
        if self.thread is None or not self.thread.is_alive():
            name = f"Cyclic send task for 0x{self.messages[0].arbitration_id:X}"
            self.thread = threading.Thread(target=self._run, name=name)
            self.thread.daemon = True
    
            if USE_WINDOWS_EVENTS:
>               win32event.SetWaitableTimer(
                    self.event.handle, 0, self.period_ms, None, None, False
                )
E               pywintypes.error: (87, 'SetWaitableTimer', 'The parameter is incorrect.')
```
</details>

--- a/can/broadcastmanager.py
+++ b/can/broadcastmanager.py
@@ -22,6 +22,7 @@ if TYPE_CHECKING:
 # try to import win32event for event-based cyclic send task (needs the pywin32 package)
 USE_WINDOWS_EVENTS = False
 try:
+    import pywintypes
     import win32event
 
     # Python 3.11 provides a more precise sleep implementation on Windows, so this is not necessary.
@@ -263,7 +264,7 @@ class ThreadBasedCyclicSendTask(
                     win32event.CREATE_WAITABLE_TIMER_HIGH_RESOLUTION,
                     win32event.TIMER_ALL_ACCESS,
                 )
-            except (AttributeError, OSError):
+            except (AttributeError, OSError, pywintypes.error):
                 self.event = win32event.CreateWaitableTimer(None, False, None)
 
         self.start()

